python/py-pew/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           github 1.0
 PortGroup           select 1.0
 
-revision            1
+revision            2
```

Comment:
I'm pretty sure that adding a subport does not require an increase in the revision - right?